kartothek/utils/store.py

```diff
@@ -181,12 +187,69 @@ def _copy_azure_cc(keys, src_store, tgt_store):
                 )
 
 
-def _copy_naive(keys, src_store, tgt_store):
-    for k in keys:
-        tgt_store.put(k, src_store.get(k))
+def _copy_naive(key_mappings, src_store, tgt_store, mapped_metadata=None):
```

Comment:
Please use https://github.com/JDASoftwareGroup/kartothek/blob/8848fd43639ea10785a7fe87833a7b6dc687850e/kartothek/utils/store.py#L189 which includes some specialised code for copy operations. For instance, it uses the copy api from azure directly which is much more efficient since it safes network traffic.